lokimq/worker.cpp

```diff
@@ -125,46 +168,66 @@ void LokiMQ::proxy_worker_message(std::vector<zmq::message_t>& parts) {
     }
     auto route = view(parts[0]), cmd = view(parts[1]);
     LMQ_TRACE("worker message from ", route);
-    assert(route.size() >= 2 && route[0] == 'w' && route[1] >= '0' && route[1] <= '9');
-    std::string_view worker_id_str{&route[1], route.size()-1}; // Chop off the leading "w"
+    assert(route.size() >= 2 && (route[0] == 'w' || route[0] == 't') && route[1] >= '0' && route[1] <= '9');
```

Comment:
these values seem pretty magical, are they documented somewhere?

Reply:
They are entirely internal for the proxy<-> worker comm layer